release.sh

```diff
@@ -411,12 +411,14 @@ if [ "$BUILD_TYPE" = "Train" ]; then
   echo
   echo "Make sure you copy notes from the deploy doc:"
   echo
-  echo "  https://docs.google.com/document/d/1lc5T1ZvQZlhXY6j1l_VMeQT9rs1mN7yYIcHbRPR2IbQ"
+  echo "### Deploy doc"
+  echo "* https://docs.google.com/document/d/1lc5T1ZvQZlhXY6j1l_VMeQT9rs1mN7yYIcHbRPR2IbQ"
```

Comment:
I don't think this one should be markup because the instruction is not to copy/paste the link itself, but instead to copy/paste the written notes from inside the deploy doc.